utils/utils.c

```diff
@@ -1065,4 +1065,57 @@ TEST_CASE(utils_strv)
 
 	return TEST_OK;
 }
+
+TEST_CASE(utils_parse_time)
+{
+	struct TC {
+		char *arg;
+		int limit;
+		uint64_t val;
+	} tc[] = {
+		{ "10.123ns", 5, 10 },
+		{ "50.987us", 5, 50987 },
+		{ "100.654ms", 5, 100654000 },
+		{ "123.123s", 5, 123123000000 },
+		{ "17.1m", 5, 1026000000000 },
+		{ "8960070.725168293s", 9, 8960070725168293 },
+		{ "8959832.082682731s", 9, 8959832082682731 },
+		{ "426883.003490100s", 9, 426883003490100 },
+		{ "8107.0641850s", 9, 8107064185000 },
+		{ "8107.006418501s", 9, 8107006418501 },
+	};
+	uint64_t time;
+	int i;
+
```

Comment:
Could you add a `pr_dbg()` to briefly describe the test like "parsing time with a unit"?

Reply:
Sure. I'll add a description.